blockapps-strato/README.md

```diff
@@ -0,0 +1,37 @@
+[![Deploy to Azure](http://azuredeploy.net/deploybutton.png)](https://azuredeploy.net/)
+
+# WARNING: THIS TEST REPOSITORY IS ONLY FOR INTERNAL MICROSOFT AND BLOCKAPPS USERS
+
+# A screencap walkthrough of the process:
+
+Click the deploy button
+
+![](./readme-images/01-deploy-button.png)
+
+Set the parameters.  Please note the resource group "blockapps"
+
+![](./readme-images/02-deploy-live-screen.png)
+
+You can also specify the "dev" image
```

Comment:
Once the new deploy to azure button code is in, the screens here will need to be updated as well